### PR TITLE
sshd: Ignore ~/.ssh/authorized_keys

### PR DESCRIPTION
**Pull request by KellerFuchs**
Will break @coltondrg's `keystore` thingie.

### CODE REVIEW

**Comment by RyanSquared** [comment]
"System Security Services"

File: ssh/sshd_config [hunk 1]
```diff
@@ -29,9 +29,9 @@ StrictModes yes
 
 RSAAuthentication yes
 PubkeyAuthentication yes
-#AuthorizedKeysFile	%h/.ssh/authorized_keys
-AuthorizedKeysCommand /usr/bin/sss_ssh_authorizedkeys
-#AuthorizedKeysCommand /usr/bin/provisor_authorized_keys
+
+AuthorizedKeysFile        none
```
Comment (DeviaVir) [inline ssh/sshd_config]: it's apparently a thing: https://lists.mindrot.org/pipermail/openssh-unix-dev/2015-July/034186.html

Comment (KellerFuchs) [inline ssh/sshd_config]: Yeah, the manpage even says so, those days  ;)

**Comment by KellerFuchs** [comment]
@singlerider As @RyanSquared said, “System Security Services”.
`/usr/bin/sss_ssh_authorizedkeys` is installed by the `sssd-common` package, and is basically a helper which queries `sssd` for users' SSH keys.  `sssd`, in turn, looks it up in LDAP (and caches the data).

**Comment by KellerFuchs** [comment]
Deployed successfully, made OpenSSH reload it's config.

**Comment by KellerFuchs** [comment]
Of course, that broke `root` access, so I had to deploy https://github.com/hashbang/shell-etc/commit/d6091e3462c854d4362afe6b872cf7a78591db8e as a hotfix...

**Pull request merged**